src/components/screens/login/networkSelector/networkSelector.js

```diff
@@ -142,7 +142,7 @@ const NetworkSelector = ({
                       onChange={onChangeInput}
                       name="customNetwork"
                       value={address}
-                      placeholder="e.g. https://mainnet-service.lisk.io or 192.168.0.1:4000"
+                      placeholder="e.g. https://mainnet-service.lisk.com or 192.168.0.1:4000"
```

Comment:
Why use mainnet-service.lisk.com here but service.lisk.com above and below? That seems inconsistent.